### PR TITLE
fix: update make deploy target to use KEPLER_IMG

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR updates `make deploy` to change `<KEPLER_IMG>` to the actual `KEPLER_IMG` value

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -242,6 +242,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 deploy: install ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | \
 		sed  -e "s|<OPERATOR_IMG>|$(OPERATOR_IMG)|g" | tee tmp/deploy.yaml | \
+		sed -e "s|<KEPLER_IMG>|$(KEPLER_IMG)|g" | tee tmp/deploy.yaml | \
```
Comment (sthaha) [inline Makefile:245]:
```suggestion
		sed  \
		  -e "s|<OPERATOR_IMG>|$(OPERATOR_IMG)|g" \
		  -e "s|<KEPLER_IMG>|$(KEPLER_IMG)|g" \
		| tee tmp/deploy.yaml  \
```

Comment (sthaha) [reply Makefile:245]: we can use a single invocation `sed`

File: Makefile [hunk 2]
```diff
@@ -241,7 +241,9 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: install ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | \
-		sed  -e "s|<OPERATOR_IMG>|$(OPERATOR_IMG)|g" | tee tmp/deploy.yaml | \
+		sed  -e "s|<OPERATOR_IMG>|$(OPERATOR_IMG)|g" \
+				 -e "s|<KEPLER_IMG>|$(KEPLER_IMG)|g" \
+		| tee tmp/deploy.yaml | \
```
Comment (sthaha) [inline Makefile:246]: spacing 🙏

Comment (vprashar2929) [reply Makefile]: my bad 😞

File: Makefile [hunk 3]
```diff
@@ -241,7 +241,9 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: install ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default | \
-		sed  -e "s|<OPERATOR_IMG>|$(OPERATOR_IMG)|g" | tee tmp/deploy.yaml | \
+		sed  -e "s|<OPERATOR_IMG>|$(OPERATOR_IMG)|g" \
+		-e "s|<KEPLER_IMG>|$(KEPLER_IMG)|g" \
```
Comment (sthaha) [inline Makefile:245]:
```suggestion
		  -e "s|<KEPLER_IMG>|$(KEPLER_IMG)|g" \
```

**Review by sthaha — APPROVED** [review]

**Pull request merged**